react/features/token-data/User.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Implements a user of conference.
+ */
+export default class User {
```

Comment:
How big of a change would it be? I'd rather make changes once, than have to do a second pass. @lyubomir WDYT? No strong opinions here.